### DISCUSSION TITLE
Is autogluon support 2x RTX 3090 multi GPU with SLI NVLINK?

### DISCUSSION THREAD

**Discussion by muazhari** [autogluon/autogluon#2747] [Q&A] [answered]
Is autogluon support 2x RTX 3090 multi GPU with SLI NVLINK? Then, which features are supported and not supported? And, in what condition do the features support multi-GPU? It would be good if you could explain how autogluon supports multi-GPU too.

**Comment by Innixma** [comment, marked as answer]
Yes AutoGluon should support this, you may want to focus on the [autogluon.multimodal tutorials](https://auto.gluon.ai/stable/tutorials/multimodal/index.html). Multi-GPU has not been thoroughly tested for autogluon.Tabular, but you can give it a try.
